Something/src/main/java/org/example/Menu.java

```diff
@@ -0,0 +1,46 @@
+package org.example;
+
+import java.util.*;
+
+public class Menu {
+   /* String placeholder;*/
+    Map<String, MenuDetails> menuMap = new LinkedHashMap<String, MenuDetails>();
+    List<String> isTrue = new ArrayList<>();
+
+    public Map<String, MenuDetails> getMenu() {
+        return menuMap;
+    }
+
+    public void setMenu(Map<String, MenuDetails> menu) {
+        this.menuMap = menu;
+    }
+
+    public List<String> getVisibleItems(){
+        List<String> visibleItems = new ArrayList<>();
+        for(MenuDetails i : menuMap.values()){
+            if(i.getVisible()){
+                System.out.println(i.getMenuTitle());
+                visibleItems.add(i.getMenuTitle());
```

Comment:
System.out.println is useful when debugging code, but when you check your code in, its usually expected that they will be removed.